locales/templates/app.pot

```diff
@@ -1237,6 +1237,11 @@ msgctxt "news_update_btn"
 msgid "New update"
 msgstr ""
 
+#. Description of service provided by which parnter
```

Comment:
Nit: should be "partner". 
Probably better to phrase this as something along the lines of "Description of service provided by a given partner"
I know it doesn't seem much, but I get afraid some non-native English speakers may get confused :) 
thanks!